### PR TITLE
[UT]Fix the failure due to the changing of JavaScript interfaces.

### LINKED ISSUE #39: Only 1 test passed
## Issue description
I tried to clone the repository and run `npm install` and then `npm test` but got only 1 test passed.
Notice: ros2-related native libraries already have been installed.

## Steps to reproduce
1. Install ros2 by following [this guide](https://github.com/ros2/ros2/wiki/Linux-Development-Setup)
2. `git clone https://github.com/RobotWebTools/rclnodejs.git && cd rclnodejs && npm install`
3. `npm install -g mocha && npm test`

The I got this:
```

> rclnodejs@0.0.1 test /home/qiuzhong/Workspace/Work/ros2/robotwebtools/rclnodejs
> mocha



  rclnodejs node test suite
    ✓ Try creating a node

  rclnodejs publisher test suite
    1) Try creating a publisher
    2) Try publish a message

  ROSIDL Node.js message generator test suite
    3) Try require all message classes
    4) Try use stdmsgs/msg/String.msg, using getMessageClass() function overrides
    5) Testing message with all-primitive members - ColorRGBA
    6) Testing copy-constructor - Duration
    7) Testing assignment of an all-primitive message - Time
    8) Testing a compound message - Pose
    9) Testing constants - GoalStatus
    10) Testing array - Int32


  1 passing (963ms)
  10 failing

  1) rclnodejs publisher test suite Try creating a publisher:
     TypeError: Cannot read property 'getMessageType' of undefined
      at Context.<anonymous> (test/test-publisher.js:33:32)

  2) rclnodejs publisher test suite Try publish a message:
     TypeError: Cannot read property 'getMessageType' of undefined
      at Context.<anonymous> (test/test-publisher.js:40:32)

  3) ROSIDL Node.js message generator test suite Try require all message classes:
     TypeError: Cannot read property 'getMessageClass' of undefined
      at msgTypeList.forEach (test/test-rosidl-message-generator.js:37:35)
      at Array.forEach (native)
      at Context.<anonymous> (test/test-rosidl-message-generator.js:36:17)

  4) ROSIDL Node.js message generator test suite Try use stdmsgs/msg/String.msg, using getMessageClass() function overrides:
     TypeError: Cannot read property 'getMessageClass' of undefined
      at Context.<anonymous> (test/test-rosidl-message-generator.js:42:34)

  5) ROSIDL Node.js message generator test suite Testing message with all-primitive members - ColorRGBA:
     TypeError: Cannot read property 'getMessageClass' of undefined
      at Context.<anonymous> (test/test-rosidl-message-generator.js:62:33)

  6) ROSIDL Node.js message generator test suite Testing copy-constructor - Duration:
     TypeError: Cannot read property 'getMessageClass' of undefined
      at Context.<anonymous> (test/test-rosidl-message-generator.js:76:33)

  7) ROSIDL Node.js message generator test suite Testing assignment of an all-primitive message - Time:
     TypeError: Cannot read property 'getMessageClass' of undefined
      at Context.<anonymous> (test/test-rosidl-message-generator.js:97:33)

  8) ROSIDL Node.js message generator test suite Testing a compound message - Pose:
     TypeError: Cannot read property 'getMessageClass' of undefined
      at Context.<anonymous> (test/test-rosidl-message-generator.js:119:33)

  9) ROSIDL Node.js message generator test suite Testing constants - GoalStatus:
     TypeError: Cannot read property 'getMessageClass' of undefined
      at Context.<anonymous> (test/test-rosidl-message-generator.js:182:33)

  10) ROSIDL Node.js message generator test suite Testing array - Int32:
     TypeError: Cannot read property 'getMessageClass' of undefined
      at Context.<anonymous> (test/test-rosidl-message-generator.js:220:33)



npm ERR! Test failed.  See above for more details.

```

### PR DESCRIPTION
**Pull request by minggangw**
Fix #39

### CODE REVIEW

**Comment by minggangw** [comment]
@kenny-y PTAL thanks!

File: test/test-rosidl-message-generator.js [hunk 1]
```diff
@@ -28,20 +28,9 @@ describe('ROSIDL Node.js message generator test suite', function(){
...
-    msgTypeList.forEach((msgType) => {
-      const MessageClass = message.getMessageClass(msgType);
-    });
-  });
-
-  it('Try use stdmsgs/msg/String.msg, using getMessageClass() function overrides', function() {
-    const MessageClass1 = message.getMessageClass(message.getMessageType('std_msgs', 'msg', 'String'));
-    assert.equal(MessageClass1.name, 'std_msgs__msg__String');
-    let msg = new MessageClass1();
+  it('Try use std_msgs/msg/String.msg', function() {
```
Comment (kenny-y) [inline test/test-rosidl-message-generator.js:49]: I think we need to put a complete list here, and try require every message class in the list, to make sure at least the message class generation is OK

Comment (minggangw) [reply test/test-rosidl-message-generator.js:49]: Sure, I will add the original logic.

**Comment by minggangw** [comment]
@kenny-y I added an ut per comment, PTAL.

**Review by kenny-y — APPROVED** [review]

**Pull request merged**